ClangSharpPInvokeGenerator/Generated.Custom.cs

```diff
@@ -117,30 +117,38 @@ public static IntPtr codeCompleteAt(CXTranslationUnit @TU, string @complete_file
         public static int indexSourceFile(CXIndexAction @param0, CXClientData @client_data, IndexerCallbacks[] @index_callbacks, uint @index_callbacks_size, uint @index_options, string @source_filename, string[] @command_line_args, int @num_command_line_args, CXUnsavedFile[] @unsaved_files, uint @num_unsaved_files, out CXTranslationUnit @out_TU, uint @TU_options)
         {
             var arr = new _CXUnsavedFile[unsaved_files.Length];
+            IntPtr result = IntPtr.Zero;
 
             try
             {
                 BeginCXUnsavedFileMarshal(ref arr, ref unsaved_files);
-                return indexSourceFile(param0, client_data, index_callbacks, index_callbacks_size, index_options, source_filename, command_line_args, num_command_line_args, arr, num_unsaved_files, out out_TU, TU_options);
+                BeginIndexerCallbacksMarshal(out result, ref index_callbacks);
```

Comment:
I'm assuming this was a bug in the original code and you're having to marshal this correctly?

Reply:
This whole commit is simply the result of copying `Generated.cs` and `Generated.Custom.cs` from the root over the top of  `ClangSharpPInvokeGenerator\Generated.cs` and `ClangSharpPInvokeGenerator\Generated.Custom.cs`.

Just making them consistent. Happy to revert this whole commit if you'd prefer?

Reply:
Thinking about this more, this is probably my mistake, I probably synced the files in the incorrect direction....

Reply:
Maybe the two of the file should be removed from `Git` and copied as part of the build/post build process? (Assuming `VS` plays nice with the missing files)

Unless I'm missing something and there's a reason they're different.